### DISCUSSION TITLE
Is there a limit to custom storage account names using AZURITE_ACCOUNTS environment variable

### DISCUSSION THREAD

**Discussion by uvproj** [Azure/Azurite#2434] [Q&A]
I am using AZURITE_ACCOUNTS environment variable to stand up multiple storage accounts for emulation and testing. Is there a limit on the number of storage accounts there?

**Comment by blueww** [comment]
There's not limitation in Azurite code, we have tested 3-4 accounts works well.
However, we have not tested with more accounts.
